src/Story.js

```diff
@@ -1,21 +1,25 @@
 import React, { Component } from 'react';
+import request from 'superagent'
 
 class Story extends Component {
     constructor(props){
       super(props);
+      this.state = {"vote": this.props.votes};
+      this.sendVote = this.sendVote.bind(this)
     }
 
-    // sendVote(){
-    //   fetch('https://api.waffle.io/cards/5825a9ed9a9ad41e008b1570', {
-    //       method: 'PUT'
-    //     })
-    //     .then((response) => {
-    //     return response.json()
-    //   }).then((stories) =>{
-    //     this.setState({'stories': stories, 'currentVote': stories[0]})
-    //   })
-    //   console.log("I HAVE VOTED")
-    // }
+    sendVote(vote){
+      alert("Thanks for your vote")
+      let voteParams = vote.toString()
+      let storyId = this.props.id
+      this.setState({"vote": voteParams})
+      request.put('http://master.bass-seahorse-cod.app.push.drieapp.co/stories/' + storyId)
```

Comment:
if we're sending votes then it should be a POST against `/stories/:story_id/vote`  I think the above PUT will set the story, which is cool too, but we should update the PR title to "PUT request to stories to change size through the UI"